### PR TITLE
Makes the heretic worm use the blood walk element

### PR DESCRIPTION
**Pull request by SuperNovaa41**
It was using a custom proc for this, but we have an element just for
this.

Code improvement, no player facing changes

### CODE REVIEW

File: code/modules/mob/living/simple_animal/heretic_monsters.dm [hunk 1]
```diff
@@ -178,6 +178,7 @@
  */
 /mob/living/simple_animal/hostile/heretic_summon/armsy/Initialize(mapload, spawn_bodyparts = TRUE, worm_length = 6)
 	. = ..()
+	AddElement(/datum/element/blood_walk)
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/simple_animal/heretic_monsters.dm:181]:
```suggestion
	AddElement(/datum/element/blood_walk, /obj/effect/decal/cleanable/blood/tracks)
```

File: code/modules/mob/living/simple_animal/heretic_monsters.dm [hunk 2]
```diff
@@ -261,21 +261,14 @@
...
-	decal.setDir(dir)
-
 /mob/living/simple_animal/hostile/heretic_summon/armsy/Destroy()
 	if(front)
 		front.icon_state = "armsy_end"
 		front.icon_living = "armsy_end"
 		front.back = null
 	if(back)
 		QDEL_NULL(back) // chain destruction baby
+	RemoveElement(/datum/element/blood_walk)
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/simple_animal/heretic_monsters.dm:271]: The Element is ELEMENT_DETACH

Comment (SuperNovaa41) [reply code/modules/mob/living/simple_animal/heretic_monsters.dm:271]: oh right i forgot how that worked

File: code/modules/mob/living/simple_animal/heretic_monsters.dm [hunk 3]
```diff
@@ -178,6 +178,7 @@
  */
 /mob/living/simple_animal/hostile/heretic_summon/armsy/Initialize(mapload, spawn_bodyparts = TRUE, worm_length = 6)
 	. = ..()
+	AddElement(/datum/element/blood_walk, /obj/effect/decal/cleanable/blood/tracks, target_dir_change = TRUE)
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/heretic_monsters.dm:181]: you want to add this after spawn_bodyparts returns, as we only want the front to do gib trails

Comment (SuperNovaa41) [reply code/modules/mob/living/simple_animal/heretic_monsters.dm:181]: gotcha

**Pull request merged**